abs.yaml

```diff
@@ -0,0 +1,5 @@
+upload_channels:
```

Comment:
why not upload to default ?

Reply:
It depends on `daal4py` https://github.com/AnacondaRecipes/daal4py-feedstock/pull/10 and -> `mpich` https://github.com/AnacondaRecipes/mpich-feedstock/pull/6